functions/go/apply-setters/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	kyaml "sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+//nolint
+func main() {
+	resourceList := &framework.ResourceList{}
+	resourceList.FunctionConfig = map[string]interface{}{}
+
+	cmd := framework.Command(resourceList, func() error {
+		s, err := getSetters(resourceList.FunctionConfig)
+		if err != nil {
+			return fmt.Errorf("failed to get data field from function config: %w", err)
```

Comment:
Failure messages need to be much more specific.

`data` field is missing in function config of kind `ConfigMap`

Please do an audit and make sure all error messages are succinct and precise.

Reply:
As users don't see `data` field at all, modified the error message without mentioning that.